[Bug]: This delay value is unsupported: 1d

### What happened?

Setting the `Time required before considered idle` to 1440 (1 day) causes the error.

![image](https://github.com/Azn9/JetBrains-Discord-Integration/assets/74133927/ca8abafa-13a9-47e2-a963-58ba43ca2d5b)

### IDE version

PyCharm 2023.3.5 (Professional Edition)

### Plugin version

2.0.8

### What operating system are you using?

Windows

### Relevant log output

```shell
com.intellij.diagnostic.PluginException: This delay value is unsupported: 1d [Plugin: dev.azn9.plugins.discord]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:99)
	at com.intellij.ide.IdleTracker.checkDelay-VtjQ1oo(IdleTracker.kt:81)
	at com.intellij.ide.IdleTracker.addIdleListener(IdleTracker.kt:58)
	at dev.azn9.plugins.discord.time.TimeService.load(TimeServiceAbstract.kt:133)
	at dev.azn9.plugins.discord.postLoad.TimePreloadingActivity$execute$2.invokeSuspend(TimePreloadingActivity.kt:34)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
Caused by: java.lang.IllegalArgumentException: This delay value is unsupported: 1d
	... 10 more
```

--- a/plugin/common/src/main/kotlin/dev/azn9/plugins/discord/settings/impl/ApplicationSettingsImpl.kt
+++ b/plugin/common/src/main/kotlin/dev/azn9/plugins/discord/settings/impl/ApplicationSettingsImpl.kt
@@ -36,7 +36,7 @@ class ApplicationSettingsImpl : ApplicationSettings, PersistentStateOptionHolder
         "Time required before considered idle",
         "Time without any activity before the plugin marks the session as idle. Changes mighty require a restart to take effect",
         5,
-        1..24 * 60,
+        1 until 24 * 60,
         format = "# " + "Minutes"
     )
     override val timeoutResetTimeEnabled by timeoutOptionPair.second.check("Reset open time when returning", "Reset open time for the application as well as open projects and files", true)

